javascript.md

```diff
@@ -1379,8 +80,6 @@ mainmenu: true
   - [innerHTML vs textContent for script text](http://jsperf.com/innerhtml-vs-textcontent-for-script-text)
   - [Long String Concatenation](http://jsperf.com/ya-string-concat)
 
-
-
 ## Resources
```

Comment:
Do we want to keep this section as well as the Performance section? Are they still relevant?